modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -75,6 +75,10 @@ ComputeMultipleInelasticStress::ComputeMultipleInelasticStress(const InputParame
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(getMaterialPropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
     _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _has_compliance(hasMaterialProperty<RankFourTensor>(_base_name + "compliance_tensor")),
```

Comment:
No, he can **not**. He can not get the material property after construction. And `hasMaterialProperty` will not return the correct (expected) result if he is using the `getDefaultMaterialProperty` call.